### ISSUE TITLE
Mermaid problem: the class attributes will be hidden with both namespace and relationship defined

### ISSUE THREAD

**Issue by bioinformatist** [go-gitea/gitea#26398]
## Code for the diagram

Let me show a simple class diagram:

```mermaid
classDiagram
namespace a-workspace {
        class A {
            SomeType a_field
        }
        class B {
            SomeType another_field
        }
        class Parse{
            <<trait>>
            parse(&mut self) TheThirdClass
        }
    }
    Parse <|.. A
    Parse <|.. B
```

Obviously it can not be rendered on GitHub.

## Result in live editor

It works well in official [live editor](https://mermaid.live):

![image](https://github.com/go-gitea/gitea/assets/17194719/5baeac0a-c9f6-4e8c-9d87-aa6ac6708454)

## Result on Gitea

All class attributes will be hidden with both namespace and relationship defined:

![image](https://github.com/go-gitea/gitea/assets/17194719/89338a56-dca6-4e74-80f8-aa26f8ec9ce4)

## With only namespace while without relationship

But it do work if the relationshop was removed:

```mermaid
classDiagram
namespace a-workspace {
        class A {
            SomeType a_field
        }
        class B {
            SomeType another_field
        }
        class Parse{
            <<trait>>
            parse(&mut self) TheThirdClass
        }
    }
```

Gitea renders right image this time, while GitHub fails again.

![image](https://github.com/go-gitea/gitea/assets/17194719/6d6e9460-b4c2-43a4-aaa0-67b7c5476426)

### Gitea Version

1.20.2

### Can you reproduce the bug on the Gitea demo site?

Yes

### How are you running Gitea?

The Gitea instance we used was deployed in the way of "[Installation with Docker](https://docs.gitea.com/installation/install-with-docker)".

### Database

PostgreSQL

**Comment by bioinformatist** [comment]
@silverwind Any news, bro?:smile:

**Comment by lng2020** [comment]
> @silverwind Any news, bro?😄

![image](https://github.com/go-gitea/gitea/assets/70063547/04eb354e-9c01-4fe1-885d-451e59013318)
already fix this one

**Comment by bioinformatist** [comment]
LGTM, let me have a try, thanks!

**Comment by lng2020** [comment]
is it okay to close it?

**Comment by bioinformatist** [comment]
It seems version 1.20.4 still has this problem, but it has been fixed in 1.21 (by test in official demo), thanks!
